Tests/Functional/CompositeRepositoryTest.php

```diff
@@ -13,6 +13,8 @@
 
 class CompositeRepositoryTest extends PhpcrRepositoryTestCase
```

Comment:
except from the setUp the abstract class is not related to PHPCR i just whanted to force to an equal test behavior as far as possible. I think as the repositories have same interfaces, so the test should have too.